### DISCUSSION TITLE
can't flash: Detected overlap at address: 0x8000 for file: partitions.bin

### DISCUSSION THREAD

**Discussion by bzzz77** [emsesp/EMS-ESP32#1562] [Q&A]
I was going to upgrade from 3.4.4, grabbed all the bins as the documentation suggests:
`$ esptool --chip esp32 --port /dev/ttyACM1 --baud 921600 --before default_reset --after hard_reset write_flash -z --flash_mode dio --flash_freq 40m --flash_size detect 0x1000 bootloader_dio_40m.bin 0x8000 partitions.bin 0xe000 boot_app0.bin 0x10000  EMS-ESP-3_6_4-ESP32.bin 
usage: esptool write_flash [-h] [--erase-all] [--flash_freq {keep,80m,60m,48m,40m,30m,26m,24m,20m,16m,15m,12m}]
                           [--flash_mode {keep,qio,qout,dio,dout}]
                           [--flash_size {detect,keep,256KB,512KB,1MB,2MB,2MB-c1,4MB,4MB-c1,8MB,16MB,32MB,64MB,128MB}]
                           [--spi-connection SPI_CONNECTION] [--no-progress] [--verify] [--encrypt]
                           [--encrypt-files <address> <filename> [<address> <filename> ...]]
                           [--ignore-flash-encryption-efuse-setting] [--force] [--compress | --no-compress]
                           <address> <filename> [<address> <filename> ...]
esptool write_flash: error: argument <address> <filename>: Detected overlap at address: 0x8000 for file: partitions.bin`

**Comment by proddy** [comment]
I think that the command line needs to be updated since > 3.4, we've changed the partition sizes. Use the EMS-ESP Flasher as that's guaranteed to work, until we've looked and revised the esptool options

**Reply by proddy** [reply]
I've updated the doc with the correct esptool command. thanks for pointing this out.

**Comment by bzzz77** [comment]
if I got it correctly, you've replaced bootloader_dio_40m.bin with bootloader.bin, but the latter is not available in https://github.com/emsesp/EMS-ESP32/tree/main/scripts

**Reply by proddy** [reply]
it's in the dev branch, https://github.com/emsesp/EMS-ESP32/blob/dev/scripts/bootloader.bin

**Comment by bzzz77** [comment]
thanks for your help.
